__tests__/fake-app.ts

```diff
@@ -0,0 +1,142 @@
+'use-strict';
+
+export function onSync(body) {
```

Comment:
Is there a reason not to import `actions-on-google`, construct an `app` constant, then export that? You could even export it as a `default`.